onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -2313,10 +2313,18 @@ def aten_fix(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_flip(self: TensorType, dims: Sequence[int]) -> TensorType:
+@torch_op("aten::flip")
+def aten_flip(self: TTensor, dims: Sequence[int]) -> TTensor:
```

Comment:
```suggestion
def aten_flip(self: TTensor, dims: INT64) -> TTensor:
```

i would take dims as INT64

Reply:
The original test case defined: self is input, dims is kwargs {"dims": (0,1,2)}, it is about dimension list, not a tensor.
[Here](https://pytorch.org/docs/master/generated/torch.flip.html?highlight=flip#torch.flip) shows the torch document, like this:
```
torch.flip(x, [0, 1])
```
So I think "attribute" is more appropriate.